src/components/RelatedVideosList/RelatedVideosList.jsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 import RelatedVideoItem from '../RelatedVideoItem/RelatedVideoItem';
-import { ListWrapper } from './styled';
+import { ListWrapper, Title, Message } from './styled';
 
-function RelatedVideosList({ videos }) {
-  console.log('----', videos);
+function RelatedVideosList({ videos, title }) {
   let related = videos
```

Comment:
Why did you use `let`? If the value is not going to change, it is recommended to use `const` so we keep close to the functions without side effects